packages/reporter/src/commands/command.tsx

```diff
@@ -402,7 +401,15 @@ class Command extends Component<Props> {
           {this._children()}
         </li>
         {model.err?.isRecovered && (
-          <li><TestError err={model.err} testId={model.testId} commandId={model.id} groupLevel={groupLevel}/></li>
+          <li>
+            <TestError
+              err={model.err}
+              testId={model.testId}
+              commandId={model.id}
+              // if the err is recovered and the current command is a log group, nest the test error within the group
+              groupLevel={this.props.groupId && groupLevel === this.props.groupId ? ++groupLevel : groupLevel}
```

Comment:
Meant to ensures `reject false` case has the correct group in a recreated session and will collapse when the group is closed, but looks like this logic is broken. Meant to fix this case (whoops!)
<img width="629" alt="Screen Shot 2022-11-02 at 4 33 20 PM" src="https://user-images.githubusercontent.com/14099737/199606761-2279d14b-9808-4b26-807d-4089ff216b13.png">